src/styles/blocks/feedback.css

```diff
@@ -0,0 +1,28 @@
+.feedback {
+  display: flex;
+  justify-content: space-between;
+}
+
+.feedback__stars {
+  display: flex;
+  margin-bottom: 24px;
+}
+
+.stars__star {
+  height: 16px;
+  margin-right: 4px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(../../images/star.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(../../images/star-active.svg);
+}
```

Comment:
it's desirable to arrange the rating in such a way that when the number changes, the number of stars also changes.